### ISSUE TITLE
Auction status shall be updated on each block

### ISSUE THREAD

**Issue by gabmontes** [autonomoussoftware/metronome-api#5]
https://github.com/MetronomeToken/metronome-api/blob/e26d11c2846b4c4b33f61594dc5e21454b43b83f/lib/tasks.js#L19

Auction status is emitted every 10 seconds but it will change only when new blocks are mined. Therefore, the event shall be emitted only after a new block event is received.

As a stretch goal, it should be emitted only if the status changed since last block. Otherwise, no event should be emitted to save resources.
